x-pack/plugins/cases/server/authorization/utils.ts

```diff
@@ -64,3 +66,11 @@ export const includeFieldsRequiredForAuthentication = (fields?: string[]): strin
   }
   return uniq([...fields, OWNER_FIELD]);
 };
+
+export const getAuthorizedAndUnauthorizedSavedObjects = <T>(
```

Comment:
How about we rename this to `groupByAuthorization` or `partitionByAuthorization` something like that.